R/init.R

```diff
@@ -19,20 +19,28 @@
 #' @param rhino_version When using an existing `renv.lock` file,
 #' Rhino will install itself using `renv::install(rhino_version)`.
 #' You can provide this argument to use a specific version / source, e.g.`"Appsilon/rhino@v0.4.0"`.
+#' @param force Create a `rhino` app without checking for if working directory is home directory.
+#' Defaults to `FALSE` to prevent creating a `rhino` app in the home directory
+#' Set to `TRUE` to create a `rhino` app in the home directory.
 #' @return None. This function is called for side effects.
 #'
 #' @export
 init <- function(
   dir = ".",
   github_actions_ci = TRUE,
-  rhino_version = "rhino"
-) {
-  init_impl(
-    dir = dir,
-    github_actions_ci = github_actions_ci,
-    rhino_version = rhino_version,
-    new_project_wizard = FALSE
-  )
+  rhino_version = "rhino",
+  force = FALSE
+)  {
+  is_home <- is_dir_home(dir = dir, force = force)
+
+  if (!is_home) {
```

Comment:
Don't we need this check in the `init_rstudio` function as well? If not, perhaps add a comment explaining why.

Reply:
`init_rstudio()` is used when we try to build a `rhino` app using RStudio's new project wizard, right? I tried to build a `rhino` app in home, and it seems RStudio's new project wizard already prevents building a new project in the home directory.  

With new project wizard, users need to provide a directory name and specify which directory it should be a subdirectory of. Users can choose home, `~`, or directories lower than home, but they cannot go higher and choose the home directory to be their project directory.

![image](https://user-images.githubusercontent.com/110383037/196355990-f3d3d77d-bb2e-40ab-a937-cb5e1a69cadb.png)